client/js/.ViewMap.js

```diff
@@ -1,6 +1,10 @@
 module.exports={
-	 Footer: require('./views/Footer'),
```

Comment:
Shouldn't need to edit this either.

Reply:
I don't think I modified this file either.  Is this done automatically?

Reply:
Its automatic, my bad.  I noticed some indentations, must be programmatic.  Sorry for the confusion.